File: tests/fixtures/zig/sample.zig
// Module comment
const std = @import("std");

// Function comment
fn greet(name: []const u8) []const u8 {
    // Inline comment
    return "Hello";
}

// Struct comment
const Greeter = struct {
    fn greet() []const u8 {
        return "hi";
    }
};

File: tests/fixtures/py/sample.py
"""Module docstring."""
import os
import sys

# A line comment
x = 1  # inline comment

def hello(name: str) -> None:
    """Function docstring."""
    print(f"Hello, {name}")

class Greeter:
    """Class docstring."""

    def greet(self) -> str:
        # method comment
        return "hi"
